RELEASE.md

```diff
@@ -95,3 +95,36 @@ git checkout develop
 git branch -d release/$RELEASE_TAG
 ```
 
+## Hotfix 
```

Comment:
Thank you @srcansiz 

Why don't we copy and adapt the procedure for release (that was already thorouly tested) ?

I suspect we forgot several points in this hotfix procedure:
1. push the changes
2. modifying `fedbiomed/common/constants.py`
3. remove the local hotfix branch

Moreover, it introduces a difference in tagging with the way we do for the release (we tag the hotfix branch, so the tag will exist in both master and develop). This may be nice to use this way of tagging also for the release process, but in any case I would suggest to keep release and hotfix procedures coherent.

All in all, I pushed the changes for 1. 2. 3. but I would suggest to rewrite the Hotfix procedure by cut/paste/adapt of the release procedure, previously modified by tagging the way you suggest. What do you think of that ?

Reply:
Hi @srcansiz 

I pushed a new version matching this proposal.

It also includes your improvements (tag in release branch, document 2 ways of restoring the branch).

I also modified the paragraph about pushing documentation: I agree that news and documentation fixes can use hotfix, but I don't think new pages of documentation should be published this way. Usually, it should not make sense to add a new page as a hotfix: new page would be related to new functionality, or func change - then it is part of a normal PR to develop, then merged as part of a new release ?

Reply:
HI @mvesin, 

Thanks for re-writing hotfix process. 

New pages can be considered as hotfix unless it is not a page in documentation page. When there is a new news added it is actually a page. What I meant was mostly the pages like `Aboust Us`, `Publications` etc.

Reply:
Thanks @srcansiz , 

OK, then we probably need a sentence that makes clear what is admitted (from a team process point of view) to be published as a hotfix ?

From the previous documentation we may pick:
https://gitlab.inria.fr/fedbiomed/fedbiomed.gitlabpages.inria.fr/-/blob/develop/README.md#process-for-updating-news-pages-etc

```
What are the changes that will be considered as 'update'

* Adding news or updating their content
* Adding new items to front page (button, boxes, description, new section)
* Adding new items for footer area
* Adding new item to menu (not for docs)
* Changing contents of static pages About us, Contributors etc.
* typo/bugs for docs (only for current version)
* Front-End issues broken layout etc.

All the other changes, adding new documentation for the next releases etc. should be considered as feature and shouldn't be merge directly to the master. It should be merged into develop by following the rules of gitflow (MR, review etc.). Develop branch should be merged into master when we do the new release in master Fed-BioMed.
```